bot/src/stake_pool.rs

```diff
@@ -882,7 +882,10 @@ mod test {
     };
 
     fn num_stake_accounts(rpc_client: &RpcClient, authority: Pubkey) -> usize {
```

Comment:
aside: wtf?  can't we just resolve this from the big grab at the start?

Reply:
No worries, this is just for the tests!